app/views/admin/_admin_nav.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1 style="float:left; margin:20px;">Admin Dashboard</h1>
+<h3 style="float:right; background-color:lightgrey; margin:20px;"><%= link_to "Dashboard", admin_index_path, style: "padding:10px; border-style: outset;" %>
```

Comment:
I like that you added some additional formatting here! This is gonna be so pretty 🤩